udata_search_service/infrastructure/search_clients.py

```diff
@@ -339,6 +359,54 @@ def query_reuses(self, query_text: str, offset: int, page_size: int, filters: di
         res = [hit.to_dict(skip_empty=False) for hit in response.hits]
         return results_number, res
 
+    def query_dataservices(self, query_text: str, offset: int, page_size: int, filters: dict, sort: Optional[str] = None) -> Tuple[int, List[dict]]:
+        s = SearchableDataservice.search()
```

Comment:
Nit: could we possible not use single letter variables? Maybe something like `search`?